common/pinot/pinotQueryValidator_test.go

```diff
@@ -133,6 +133,22 @@ func TestValidateQuery(t *testing.T) {
 			query:     "StartTime >= 1697754674",
 			validated: "StartTime >= 1697754674",
 		},
+		"Case15-4: unix nano converts to milli seconds for equal statements": {
```

Comment:
please add test cases to cover all failure scenarios

Reply:
sure, added more test cases